### PR TITLE
Provide more consistent copy in the security token dialogue and a link to find logs

### PR DESCRIPTION
**Pull request by rtyler**
I believe this text is more clear for a new user to understand why they must
complete this step of the process.

Fixes [JENKINS-33462](https://issues.jenkins-ci.org/browse/JENKINS-33462)

jenkins2_security_token_1

### CODE REVIEW

File: core/src/main/resources/jenkins/install/SetupWizard/authenticate-security-token.jelly [hunk 1]
```diff
@@ -8,19 +8,24 @@
...
                 <div class="modal-body">
                   <i class="water-mark icon-service"></i>
                   <div class="jumbotron welcome-panel offline">
                   
-                    <h1>${%Security Token}</h1>
-                    <p>${%As an extra measure of security, please enter the setup security token to proceed.}</p>
-                    <p>${%It can be found in the logs for this Jenkins instance.}</p>
+                    <h1>${%Unlock Jenkins}</h1>
+                    <p>${%To ensure Jenkins is securely set up by the administrator, a setup security token has been printed to the logs.}
+                        (<small>
```
Comment (daniel-beck) [inline core/src/main/resources/jenkins/install/SetupWizard/authenticate-security-token.jelly]: Putting this in parentheses looks like something that works well in English but may not in some languages.

**Comment by daniel-beck** [comment]
Definitely an improvement :+1:

File: core/src/main/resources/jenkins/install/SetupWizard/authenticate-security-token.jelly (hunk 1 above)
Comment (rtyler) [inline core/src/main/resources/jenkins/install/SetupWizard/authenticate-security-token.jelly]: Good point, fixing

**Comment by kzantow** [comment]
:+1: I like it

**Comment by rtyler** [comment]
Apparently each pull request kicks off an ATH run now? Wacky

**Comment by kohsuke** [comment]
I'm running out of time. I'm merging this now

**Pull request merged**